api/src/main/java/org/pmiops/workbench/actionaudit/auditors/UserServiceAuditAdapterImpl.java

```diff
@@ -61,12 +80,13 @@ public void fireAdministrativeBypassTime(
       long userId,
       BypassTimeTargetProperty bypassTimeTargetProperty,
       Optional<Instant> bypassTime) {
+    DbUser adminUser = dbUserProvider.get();
```

Comment:
So if we had a cron that happened to set this bypass time for some reason,  we'd. have to fix this one too.

Reply:
Not likely in the short term I guess, but there's nothing baked in to prevent that, right? Like if we needed to do a batch access change because we added or removed an access level.

Not worth worrying about right now.